Python/sampleclient/filechecks.py

```diff
@@ -0,0 +1,40 @@
+import os
+from pathlib import Path
+
+
+def file_exists(file_path):
+    '''Returns if file exists or not.'''
+    file = Path(file_path)
+    return file.exists()
+
+
+def file_size_in_bytes(file_path):
+    '''Return size of file in bytes, or -1 if file does not exist.'''
+    if not file_exists(file_path):
+        return -1
```

Comment:
I thought of that, but I was thinking it would be better to be explicit; But I agree None is more pythonic than just being explicit, so its been updated/changed.